serverTest.py

```diff
@@ -2,24 +2,36 @@
 import server
 
 class TestSocketIO(unittest.TestCase):
-    def test_connect(self):
+    def test_if_connection_successful(self):

```

Comment:
This is what I'm imagining for the tests:

def test_connect_emitsboradstate - tests that the client receives the board state after a successful connection
def test_disconnect_printsdisconnectionmessage - tests that the server prints out the disconnect msg
def test_sendstroke_updatesserverboard - tests that 'send-update' from client updates the server board
def test_sendstroke_broadcastsUpdate - tests that 'send-update' from client triggers broadcast from server

Remember that you're not trying to test the socketio API itself, but rather the behavior of the server which happens to use socketio.